bundles.go

```diff
@@ -103,6 +110,57 @@ func CopyBundles(bundles []Bundle) []Bundle {
 	return copy
 }
 
+// filterSameSeriesBundles filters bundles that are from the same semver series as
+// input bundle. For major version bundle (e.g. 3.0.0) this function will return
+// all available major bundles. For minor all minor bundles from the same major bundle.
+// For patch all patch versions from the same minor bundle.
```

Comment:
In this function main logic. Depending on input bundle it outputs bundles that related to the same "series" (or same release). e.g. 3.0.1 is patch release, so all 3.0 release patch releases will be returned.